resources/app.yaml

```diff
@@ -22,7 +22,25 @@ hooks:
               - name: hook
                 image: quay.io/gravitational/debian-tall:0.0.1
                 command: ["/usr/local/bin/kubectl", "apply", "-f", "/var/lib/gravity/resources/resources.yaml"]
-
+  update:
+    spec:
+      apiVersion: batch/v1
+      kind: Job
+      metadata:
+        name: logging-app-update
+      spec:
+        parallelism: 1
+        completions: 1
+        activeDeadlineSeconds: 100
```

Comment:
This still looks a bit magic - why does only `rig status` take the changeset as parameter? Is it implicit in all other commands (or not necessary)?